### PR TITLE
paru: add pt_BR translation

### PR DESCRIPTION
**Pull request by isaacvicente**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution

**Comment by isaacvicente** [comment]
This page is still missing the imperative mood of the style guide. Can @navarroaxel suggest the changes, so I can apply them?

File: pages.pt_BR/linux/paru.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Pesquisar e instalar interativamente um pacote:
+
+`paru {{nome_do_pacote_ou_termo_de_pesquisa}}`
+
+- Sincronizar e atualizar todos os pacotes:
+
+`paru`
+
+- Atualizar pacotes do AUR:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/paru.md:14]:
```suggestion
- Atualiza pacotes do AUR:
```

File: pages.pt_BR/linux/paru.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# paru
+
+> Um auxiliar do AUR e um wrapper do pacman.
+> Mais informações: <https://github.com/Morganamilo/paru>.
+
+- Pesquisar e instalar interativamente um pacote:
+
+`paru {{nome_do_pacote_ou_termo_de_pesquisa}}`
+
+- Sincronizar e atualizar todos os pacotes:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/paru.md:10]:
```suggestion
- Sincroniza e atualizar todos os pacotes:
```

File: pages.pt_BR/linux/paru.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# paru
+
+> Um auxiliar do AUR e um wrapper do pacman.
+> Mais informações: <https://github.com/Morganamilo/paru>.
+
+- Pesquisar e instalar interativamente um pacote:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/paru.md:6]:
```suggestion
- Pesquisa e instala interativamente um pacote:
```

File: pages.pt_BR/linux/paru.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Sincronizar e atualizar todos os pacotes:
+
+`paru`
+
+- Atualizar pacotes do AUR:
+
+`paru -Sua`
+
+- Obter informações sobre um pacote:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/paru.md:18]:
```suggestion
- Obtém informações sobre um pacote:
```

File: pages.pt_BR/linux/paru.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`paru -Si {{nome_do_pacote}}`
+
+- Fazer o download do `PKGBUILD` e outros arquivos de origem do pacote do AUR ou ABS:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/paru.md:22]:
```suggestion
- Faz o download do `PKGBUILD` e outros arquivos de origem do pacote do AUR ou ABS:
```

File: pages.pt_BR/linux/paru.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`paru --getpkgbuild {{nome_do_pacote}}`
+
+- Exibir o arquivo `PKGBUILD` de um pacote:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/paru.md:26]:
```suggestion
- Exibe o arquivo `PKGBUILD` de um pacote:
```

**Comment by isaacvicente** [comment]
Thanks @kbdharun and @navarroaxel for the help. Soon I hope I don't be a "noob" in the PRs.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**